doc/examples/config/config-storage/README.md

```diff
@@ -313,6 +319,11 @@ config.storage.txn({
     is_success: false
   revision: 10
 ...
+config.storage.txn({
+    predicates = {{'revision', '==', revision}},
+    on_success = {{'put', '/a', 'v2', {ttl = 60}}}
+    on_failure = {{'put', '/b', 'v1', {ttl = 30}}}
+})
```

Comment:
Two previous examples are accompanied with a comment about an idea of the exapmle. Let's do it here too.

Reply:
Updated.